client/src/pages/admin/AdminBooks.tsx

```diff
@@ -110,7 +110,7 @@ const data: BookViewModelType[] = [
     isbn: '124343',
     yearPublished: 2019,
     bookCategories: [],
-    title: 'Biblia Skurwysynu',
```

Comment:
Jakubie jeśli promotor to zobaczy...

Reply:
Liczę iż zgubi się to w gąszczu commitów